lib/scholar/metrics/neighbors.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Scholar.Metrics.Neighbors do
+  import Nx.Defn
+
+  deftransform recall(graph_true, graph_pred) do
```

Comment:
I believe these conditionals would work inside `defn`, no?

Reply:
This one won't work
```
if Nx.shape(neighbors_true) != Nx.shape(neighbors_pred) do
```
as shape tuples cannot be converted to tensors. So either I leave it like this or compare each axis separately.